helm/hpcc/templates/_helpers.tpl

```diff
@@ -615,3 +615,41 @@ resources:
     memory: "100M"
 {{- end -}}
 
+
+{{/*
+Create placement related settings
```

Comment:
it's good to add a comment here, as with other helper functions,
to indicate what's expected to be passed in in dict, e.g.:
Pass in dict with placement